homeassistant/components/arris_tg2492lg/device_tracker.py

```diff
@@ -0,0 +1,74 @@
+"""Support for Arris TG2492LG router."""
+import logging
+from typing import List
+
+from arris_tg2492lg import ConnectBox, Device
+import voluptuous as vol
+
+from homeassistant.components.device_tracker import (
+    DOMAIN,
+    PLATFORM_SCHEMA,
+    DeviceScanner,
+)
+from homeassistant.const import CONF_HOST, CONF_PASSWORD
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_IP = "http://192.168.178.1"
```

Comment:
```suggestion
DEFAULT_HOST = "http://192.168.178.1"
```

Technically this is not a host... but a URI or URL.
It is always HTTP right? So why not ask for just the host and add the protocol schema later?